### DISCUSSION TITLE
Github education benefits

### DISCUSSION THREAD

**Discussion by amekinya** [community/community#132947] [GitHub Education]
### Hi everyone,

Hello I have not recieved my education benefits within 5 days. Please help :(

**Comment by SamuelOUS** [comment]
Hi Amekinya!
Has your application been approved? The process usually doesn't take that long. If it has been approved, you should try applying again :)

**Reply by SamuelOUS** [reply]
Hey shaunechua :),

How long has your application been approved? If it was approved in the last 2 days, don't worry; it usually takes that long or an additional day at most. If it’s been longer, you should submit another request

**Comment by davevad93** [comment]
Hi @amekinya ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time can vary depending on different factors, e.g. the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @amekinya was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470
